protected/humhub/modules/user/controllers/AuthController.php

```diff
@@ -103,6 +103,10 @@ public function actionLogin()
             }
         }
 
+        if (Yii::$app->settings->get('maintenanceMode')) {
+            Yii::$app->session->setFlash('error', Yii::t('error', 'Only admins have an access to the site on maintenance mode.'));
```

Comment:
You can remove `an` it's not needed

Reply:
Commit https://github.com/humhub/humhub/pull/4773/commits/23a9ed3c0a5750e569d9480669fc7fe0d6f52ae1